scm-webapp/src/main/java/sonia/scm/security/JwtAccessTokenBuilder.java

```diff
@@ -156,7 +156,7 @@ private String getSubject(){
   @Override
   public JwtAccessToken build() {
     if (SecurityUtils.getSubject().getPrincipals().getRealmNames().contains(ApiKeyRealm.NAME)) {
-      throw new AuthorizationException("Cannot create access token for api keys");
+      scope = Scope.valueOf(SecurityUtils.getSubject().getPrincipals().oneByType(Scope.class));
     }
```

Comment:
Is this only relevant for api keys? What about a scoped JWT? Can i get more privileges, if i create a new JWT with a scoped JWT? Perhaps we should always use scope from the principals, if available?

@pfeuffer What do you think?

Reply:
Sounds reasonable. If this is as simple as here, the `if` could be replaced with a check whether we have a `Scope` principal.